rules/windows/command_and_control_rdp_tunnel_plink.toml

```diff
@@ -46,10 +47,16 @@ with tools that perform tunneling.
 
 - Initiate the incident response process based on the outcome of the triage.
 - Isolate the involved host to prevent further post-compromise behavior.
-- Disable the involved accounts, or restrict their ability to log on remotely.
-- Reset passwords for the user account and other potentially compromised accounts (email, services, CRMs, etc.).
-- Take actions to disable the tunneling.
-- Investigate the initial attack vector.
+- Take the needed actions to disable the tunneling, which can be a process kill, service deletion, registry key
+modification, etc. Inspect the host to know which method was used and the needed action for the case.
```

Comment:
```suggestion
modification, etc. Inspect the host to learn which method was used and to determine a response for the case.
```

Reply:
Made some minor language changes here. Feel free to ignore if they're incorrect!

Reply:
Nice one